acronyms.tex

```diff
@@ -48,6 +51,7 @@
 NIST & National Institute of Standards and Technology (USA) \\\hline
 NSF & National Science Foundation \\\hline
 NVMe & Non Volatile Memory Express \\\hline
+OGA & Other Government Agencies \\\hline
```

Comment:
I thought we shouldn't be using this anymore.

Reply:
this file should not be there and is not used  - i removed it.   But OGA was in the glossary and glossary was not excluded from  the files used to generate the glossary so it would always stay there.  I fixed that also in Makefile. OGA is nomore in the docuemnt.